dz-8/src/main/java/org/example/StudentsGroup.java

```diff
@@ -0,0 +1,39 @@
+package org.example;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class StudentsGroup {
+
+
+    private Student monitor;
+    private Map<Integer, Student> students;
+    private Map<Integer, Task> tasks;
+    private Map<Integer, ArrayList<Integer>> completedTasksIdsByUserIds;
+
+    StudentsGroup(Student monitor) {
+        this.monitor=monitor;
+        this.addStudent(monitor);
+    }
+    public void setMonitor(Student monitor) {
+        this.monitor = monitor;
+    }
+    public void addStudent(Student student) {
+        this.students.put(student.getId(), student);
+    }
+    public void removeStudent(Integer studentId) {
+        this.students.remove(studentId);
```

Comment:
What will be if we delete student which is head of the group, I mean `monitor`? THis behavior should be processed